src/components/page/ResultPage.js

```diff
@@ -115,7 +112,7 @@ class Result extends React.Component {
     render() {
         const {title, description} = this.state;
         const answers = this.decodeAnswer(new URLSearchParams(this.props.location.search).get("answers"));
-        const score = answers.filter((answer) => {return answer.Selected==answerList[answer.Id-1].Answer}).length;
+        const score = answers.filter((answer) => {return answer[1]==answerList[answer[0]-1].Answer}).length;
```

Comment:
Can you explain why this array is fixed?

Reply:
answer was an object and It would make url longly even if answer has only id and selected. so I have changed object to array(answer[0]= id, answer[1]= selected)

Reply:
I see. what about id and answer form way simpler? like [[1,2,3],[1,2,3]] ([[ids],[answers]])??

Reply:
the way you suggested reduced url length by 20 more than before.
result:
http://localhost:3000/result?answers=W1szMywzOSwxOCwyMywxNiwzOCwxNywyOSw0MiwxM10sWzAsMCwwLDAsMCwwLDAsMCwwLDBdXQ

Reply:
still long tho..
Let's finish it up here and discuss something more about it in other page.